packages/app/src/containers/PointControlsInspector.tsx

```diff
@@ -55,7 +55,11 @@ function getCurveModeString(value: Sketch.CurveMode): CurveModeOption {
 export default memo(function PointControlsInspector() {
   const [state, dispatch] = useApplicationState();
 
-  const points = Selectors.getSelectedPoints(state);
+  let points = Selectors.getSelectedPoints(state);
+  if (points.length === 0) {
+    const controlPoint = Selectors.getSelectedControlPoint(state);
+    controlPoint ? points.push(controlPoint) : (points = []);
```

Comment:
Outside of immer's `produce`, we should avoid methods that mutate arrays like `push` when possible. We should also avoid assignments in ternary expressions. Maybe:

```ts
const controlPoint = Selectors.getSelectedControlPoint(state);
const points = controlPoint ? [controlPoint] : Selectors.getSelectedPoints(state)
```